cdk/lib/soft-opt-in-consent-setter.ts

```diff
@@ -411,5 +414,38 @@ export class SoftOptInConsentSetter extends GuStack {
 				reason: 'Retaining a stateful resource previously defined in YAML',
 			});
 		});
+
+		// Acquisitions Event Bus (defined in support-frontend CDK)
+		const acquisitionsEventBus = EventBus.fromEventBusArn(
+			this,
+			'AcquisitionsEventBus',
+			props.acquisitionsEventBusArn,
+		);
+
+		// Rules
+		new Rule(this, 'SoftOptInToSQSRule', {
+			description:
+				'Send all events received via support-workers onto soft opt-in SQS queue',
+			eventBus: acquisitionsEventBus,
+			eventPattern: {
+				region: ['eu-west-1'],
+				source: ['support-workers.1'],
+			},
+			targets: [
+				new SqsQueue(softOptInsQueue, {
+					message: aws_events.RuleTargetInput.fromObject({
+						subscriptionId: aws_events.EventField.fromPath(
+							'$.detail.zuoraSubscriptionNumber',
+						),
+						identityId: aws_events.EventField.fromPath('$.detail.identityId'),
+						eventType: 'Acquisition',
+						productName: aws_events.EventField.fromPath(
+							'$.detail.ZuoraProductName',
+						),
```

Comment:
I think we need to add this field here: https://github.com/guardian/support-frontend/blob/4906e5f65befeff42259862558bf4f54488cf62d/support-workers/src/main/scala/com/gu/acquisitions/AcquisitionDataRowBuilder.scala#L61-L93
Also, I notice that everywhere else in that structure is `thisWay` rather than `ThisWay` so I think we should keep that convention when adding new fields.
```suggestion
						productName: aws_events.EventField.fromPath(
							'$.detail.zuoraProductName',
						),
```